packages/disploy/src/structs/ModalActionRowComponent.ts

```diff
@@ -0,0 +1,27 @@
+import {ActionRowComponent} from "./ActionRowComponent";
+import type {App} from "../client";
+
+export class ModalActionRowComponent extends ActionRowComponent {
```

Comment:
Shouldn't be extending Base since it requires you to initiate with the App, which gives pretty bad DX.

Reply:
Resolve in 85a2bafe346c977558eefdc89bbbbb2f8f77af6c